README.md

```diff
@@ -18,15 +18,16 @@ targetcli-fb is licensed under the Apache 2.0 license. Contributions are welcome
  * Tarballs: [fedorahosted](https://fedorahosted.org/releases/t/a/targetcli-fb/)
  * Playlist of instructional screencast videos: [YouTube](https://www.youtube.com/playlist?list=PLC2C75481A3ABB067)
 
-In-repo packaging
------------------
-Packaging scripts for RPM and DEB are included, but these are to make end-user
-custom packaging easier -- distributions tend to maintain their own packaging
-scripts separately. If you run into issues with packaging, start with opening
-a bug on your distro's bug reporting system.
-
-Some people do use these scripts, so we want to keep them around. Fixes for
-any breakage you encounter are welcome.
+Packages
+--------
+targetcli-fb is packaged for a number of Linux distributions including
+RHEL, Fedora, openSuse, Arch Linux or Debian.
+
+If you need to build a custom RPM or Debian package, you can refer to
+the build scripts from
+[Fedora](http://pkgs.fedoraproject.org/cgit/rpms/targetcli.git)
+or
+[Debian](https://alioth.debian.org/plugins/scmgit/cgi-bin/gitweb.cgi?p=linux-target/targetcli-fb.git).
```

Comment:
@gonzoleeman will Fedora scripts work on SuSE or should we also include a link to SuSE-style script, for custom pkg builders?